### PR TITLE
feat/ACMS-CPC-539_Add_Photo_To_Owner

### PR DESCRIPTION
**Pull request by DmitryGrigoryev21**
JIRA: https://champlainsaintlambert.atlassian.net/browse/CPC-539?atlOrigin=eyJpIjoiMWViMmZiNDA5M2Q2NGU5Njk3ZjYzNzJjODE4MTYwNDYiLCJwIjoiaiJ9

## Context:
Front end image handling for Owner profile picture

## Changes
Profile picture displayed on Owner detail page
Clicking it will allow user to change the image

## Before and After UI (Required for UI-impacting PRs)
Before
Screen Shot 2022-10-10 at 7 34 01 PM

#After
Default
Screen Shot 2022-10-10 at 7 24 08 PM
When hovered
Screen Shot 2022-10-10 at 7 25 23 PM
Click to chose file
Screen Shot 2022-10-10 at 7 25 56 PM
New image right away
Screen Shot 2022-10-10 at 7 26 45 PM

## Dev notes
Photo entity was modified to hold base64 of image as string rather than byte[]

### CODE REVIEW

**Review by felixlabrie — APPROVED** [review]
functionality tested

File: api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js [hunk 1]
```diff
@@ -7,4 +7,48 @@ angular.module('ownerDetails')
         $http.get('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
         });
+
```
Comment (AmirHamzahMirza) [inline api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js:10]: Just add some comments on exactly how this works, the depth of it is up to you

**Review by felixlabrie — APPROVED** [review]
fixed conflict at merge

**Review by AmirHamzahMirza — APPROVED** [review]
Looks good, works correctly and doesn't seem to have any issues, would like some comments especially within the JS angular work to explain how it functions deeply

**Pull request merged**